packages/terra-application-utility/src/utility/_UtilityButton.scss

```diff
@@ -0,0 +1,98 @@
+:local {
+  .header-utility-button {
+    align-items: center;
+    background-color: var(--terra-application-header-utility-background-color, transparent);
+    border-radius: var(--terra-application-header-utility-border-radius);
+    border-width: var(--terra-application-header-utility-border-width, 0);
+    color: var(--terra-application-header-utility-color, #fff);
```

Comment:
Should the defaults be transparent & white? I am assuming these will be fine within the menu & header but would this ever be standalone for some reason?

Reply:
@neilpfeiffer